### PR TITLE
Copy js map file over if it exists

### PR DESCRIPTION
**Pull request by schloerke**
Fixes https://github.com/rstudio/bootstraplib/issues/69

### CODE REVIEW

File: R/bs_sass.R [hunk 1]
```diff
@@ -119,7 +132,7 @@ bootstrap <- function(
       htmlDependency(
         name = "bootstrap",
         version = get_exact_version(version),
-        src = dirname(out_file),
+        src = out_file_dir,
         stylesheet = basename(out_file),
         script = basename(js),
```
Comment (cpsievert) [inline R/bs_sass.R:137]: Since we're already assuming this behavior, let's be explicit about it

```suggestion
  all_files = TRUE,
```

Comment (schloerke) [inline R/bs_sass.R:137]:
```suggestion
        script = basename(js),
        all_files = TRUE,

```

File: R/bs_sass.R [hunk 2]
```diff
@@ -107,10 +107,23 @@ bootstrap <- function(
...
+    js_map_file <- paste0(js, ".map")
+    # if `version` is `"3"`, then `js` is `"bootstrap-sass/javascripts/bootstrap[.min].js"`,
+    ## which does not have a map file
+    ## so check for map file existence
+    if (file.exists(js_map_file)) {
+      file.copy(js_map_file, out_file_dir, overwrite = TRUE)
+    } else {
+      TRUE
+    }
+  })
```
Comment (schloerke) [inline R/bs_sass.R:124]: Instead, we should remove the sourcemap link in the javascript file when updating the bootstrap files in `./tools`.

If users want access to the non-minified files, they can use `minified = FALSE`.

**Review by cpsievert — APPROVED** [review]
Pls update `tools/README.md` accordingly, then LGTM

**Pull request merged**